src/containers/item-details/index.tsx

```diff
@@ -79,12 +47,14 @@ const ProductDetailsArea = ({ space = 1, className, item }: Props) => {
                 {/* <ProductCategory owner={item.owner} />
                 <ProductCollection collection={product.collection} /> */}
               </div>
-              <Button color="primary-alta" path="#">
-                Unlockable content included
-              </Button>
+              {isOwner ? (
+                <Button color="primary-alta" path={item?.image.url}>
+                  Unlockable content included
+                </Button>
+              ) : null}
               <div className="rn-bid-details">
                 <BidTab />
-                <PlaceBet item={item} />
+                {!isOwner ? <PlaceBet item={item} /> : null}
```

Comment:
This is a perfect example to use the `'condition && logic'`. I feel like nulls uglify your code a bit more than just doing nothing imho.